pages/index.tsx

```diff
@@ -75,7 +76,7 @@ const HeroPage: React.FC = () => {
         const data = await response.json();
   
         const cloudinaryPhotos : Photo[] = data.map((photo: HeroImageData) => {
-          const cloudImage : CloudinaryImage = cld.image(photo.public_id).resize(auto().width(1000));
+          const cloudImage : CloudinaryImage = cld.image(photo.public_id).resize(auto());
```

Comment:
I can't tell yet if this had an adverse effect on performance. It may have, but the gifs don't like being resized too much. I will investigate further.

Reply:
It did. Reverted in favor of resizing everything except gifs